pyshelf/search/container.py

```diff
@@ -1,31 +1,28 @@
-from elasticsearch_dsl.connections import connections
 from pyshelf.search.update_manager import UpdateManager
 from pyshelf.search.manager import Manager as SearchManager
+from urlparse import urlparse
 
 
 class Container(object):
-    def __init__(self, config, logger):
+    def __init__(self, logger, connection_string):
         self.logger = logger
-        self.config = config
-        self._hosts = None
+        self._es_index = urlparse(connection_string).path[1:]
+        self._es_host = connection_string.rsplit("/", 1)[0]
```

Comment:
This should urlparse as well.  It should return an object containing all parts.  Even if you have to manually concat the port and the host for the "es_host".  Also if the host contains the port it shouldn't just be called host.  Can call it host_and_port for all I care, but just host may be confusing.